alpine/entrypoint.sh

```diff
@@ -76,6 +76,35 @@ if [[ $STATSD_METRIC_NAMESPACE ]]; then
     sed -i -e "s/^# statsd_metric_namespace:.*$/statsd_metric_namespace: ${STATSD_METRIC_NAMESPACE}/" /opt/datadog-agent/agent/datadog.conf
 fi
 
+if [[ $MESOS_MASTER || $MESOS_SLAVE ]]; then
+    # expose supervisor as a health check
+    echo "
+[inet_http_server]
+port = 0.0.0.0:9001
+" >> /opt/datadog-agent/agent/supervisor.conf
+fi
+
+if [[ $MESOS_MASTER ]]; then
+    cp /opt/datadog-agent/agent/conf.d/mesos_master.yaml.example /opt/datadog-agent/agent/conf.d/mesos_master.yaml
+    cp /opt/datadog-agent/agent/conf.d/zk.yaml.example /opt/datadog-agent/agent/conf.d/zk.yaml
+
+    sed -i -e "s/- host: localhost/- host: leader.mesos/" /opt/datadog-agent/agent/conf.d/mesos_master.yaml
```

Comment:
`leader.mesos` is the preferred way. It always resolves to the current master whereas `master.mesos` sometime resolves to a standby master.